index.js

```diff
@@ -1,4 +1,20 @@
+/* Well! I got my game starting to run but definitely not complete. There are
+two issues I would like feedback/direction on for resubmission.
```

Comment:
Thank you for these helpful comments! I'm so glad you submitted something despite the bugs and incomplete stories. That's okay! We can meet up and noodle on these a bit and see if we can get you in the right direction for resubmission.